test/gtest/ucp/ucp_test.cc

```diff
@@ -287,7 +289,11 @@ ucs_status_t ucp_test::request_process(void *req, int worker_index, bool wait,
     while (wait &&
            (ucp_request_check_status(req) == UCS_INPROGRESS) &&
            (ucs_get_time() < deadline)) {
-        check_events(entities(), wakeup, worker_index);
+        if (entity_list.size()) {
```

Comment:
minor: entity_list.empty() (and inverse if/else)
or:
```c
        check_events(entity_list.empty() ? entities() : entity_list, wakeup, 
                     worker_index);
```

Reply:
fixed